lib/routes/Character/components/CharacterDialog/components/blocks/BlockPointCounter.tsx

```diff
@@ -53,6 +50,19 @@ export function usePointCounter(props: {
     setInternalPoints((intValue - 1).toString());
   }
 
+  function incrementMax() {
+    const intValue = parseInt(internalMaxPoints ?? "") || 0;
+    setInternalMaxPoints((intValue + 1).toString());
+  }
+
+  function decrementMax() {
+    const intValue = parseInt(internalMaxPoints ?? "") || 0;
+    if (intValue - 1 === 0) {
+      setInternalMaxPoints("0");
+    }
+    setInternalMaxPoints((intValue - 1).toString());
+  }
+
```

Comment:
I added those methods so that I could add the + and - buttons on the max circle text field

![image](https://user-images.githubusercontent.com/6224111/115559355-a0f7b000-a281-11eb-9ccf-baff9e132452.png)